transport/src/main/java/io/servicefabric/transport/protocol/protostuff/BinaryMessage.java

```diff
@@ -19,6 +21,9 @@
 	@Tag(3)
 	private byte[] data;
 
+	@Tag(4)
+	private String dataClass;
```

Comment:
@kshchepanovskyi Actually in next PR I want to remove this BinaryMessage and use following approach https://github.com/servicefabric/servicefabric/blob/cluster-api-part3/transport/src/main/java/io/servicefabric/transport/protocol/MessageSchema.java (here will rename dataClass to dataType) and generic message https://github.com/servicefabric/servicefabric/blob/cluster-api-part3/transport/src/main/java/io/servicefabric/transport/protocol/Message.java